biomage/rotate_ci/rotate_ci.py

```diff
@@ -51,13 +53,30 @@ def format_name_for_cf(repo_name):
     return repo_name.replace("_", " ").replace("-", " ").title().replace(" ", "")
 
 
+def get_ci_names(org):
+    stack_name = "biomage-ci-users"
+    path_prefix = "ci-users"
+    name_prefix = "ci-user"
+
+    if org.login != DEFAULT_ORG:
+        org_postfix = org.login
+        stack_name = f"biomage-ci-users-{org_postfix}"
+        path_prefix = f"{path_prefix}/{org_postfix}"
+        name_prefix = f"{name_prefix}-{org_postfix}"
+
+    return stack_name, path_prefix, name_prefix
+
```

Comment:
why don't you do a dictionary with org name: stack name as Pol suggested? It will look cleaner

Reply:
That would force us to hard code the organization names in the code. Would that be ok?